src/backend/InvenTree/InvenTree/settings.py

```diff
@@ -523,19 +532,30 @@
     }
 ]
 
+OAUTH2_PROVIDER = {
+    # default scopes
+    'SCOPES': oauth2_scopes,
+    # OIDC
+    'OIDC_ENABLED': True,
```

Comment:
Do we want to set the `OIDC_ENABLED` attribute based on a run-time configuration parameter e.g. in the config file or env var?

Reply:
It is; while it is enabled here, the urlpatterns are flagged. Requiring them to be explicitly be turned on to work.

I also added docs regarding the flags and activation as part of this PR.